### PR TITLE
Encode Angular HttpClient generated params in console.

### PR DESCRIPTION
**Pull request by ftkg**
See: https://github.com/angular/angular/issues/11058

Solves #1037

### CODE REVIEW

File: console/ui/src/app/console.service.ts [hunk 1]
```diff
@@ -839,14 +839,14 @@ export class ConsoleService {
   /** Delete (non-recorded) all user accounts. */
   deleteAccounts(auth_token: string): Observable<any> {
 		const urlPath = `/v2/console/account`;
```
Comment (sesposito) [inline console/ui/src/app/console.service.ts:841]: There's a few places where the generated file has indendation misalignment, this is likely due to a spaces vs tabs issues in the template.

**Review by sesposito — CHANGES_REQUESTED** [review]
Just a minor 💅  comment, otherwise LGTM

File: console/console.pb.go
Comment (sesposito) [inline console/console.pb.go:1]: Is this change needed?

Comment (ftkg) [reply console/console.pb.go:1]: Probably zero impact, but I'm using the defined `protoc-gen-go` version in `go.mod`

**Review by sesposito — APPROVED** [review]
